features/dd-sdk-android-ndk/src/main/kotlin/com/datadog/android/ndk/internal/NdkCrashReportsFeature.kt

```diff
@@ -51,7 +52,7 @@ internal class NdkCrashReportsFeature(
             NDK_CRASH_REPORTS_FOLDER
         )
         try {
-            allowThreadDiskReads {
+            allowThreadDiskWrites {
```

Comment:
I wonder why do we need to create the dirs on the main thread, maybe the owner of this code can give more info while reviewing this code. Tks !!.

Reply:
yes and no, I didn't wrote that function `allowThreadDiskRead`, I think @xgouchet did ;)

Reply:
Check line 54 and 56 ;)

Reply:
yes totally right there, I am still trying to think of a reason why we need to keep it there, do you see any ?

Reply:
yes that's the reason why I am afraid as when the signal is handle you are in a different state it is not the same as in the JVM crash state. You have so little time to do something then the process is killed so you need to be very efficient there, I think for this main reason I prefer in this case to have that folder created already.